doozer/doozerlib/distgit.py

```diff
@@ -1394,10 +1394,66 @@ def _clean_repos(self, dfp):
                 if changed:
                     dfp.add_lines_at(entry, "RUN " + new_value, replace=True)
 
-    def _mapped_image_from_member(self, image, original_parents, count):
+    def _resolve_parent(self, original_parent, dfp):
+        try:
+            self.logger.debug('Retrieving image info for image %s', original_parent)
+            cmd = f'oc image info {original_parent} -o json'
+            out, _ = exectools.cmd_assert(cmd, retries=3)
+            labels = json.loads(out)['config']['config']['Labels']
+
+            # Get the exact build NVR
+            build_nvr = f'{labels["com.redhat.component"]}-{labels["version"]}-{labels["release"]}'
+
+            # Query Brew for build info
+            self.logger.debug('Retrieving info for Brew build %s', build_nvr)
+            with self.runtime.shared_koji_client_session() as koji_api:
+                if not koji_api.logged_in:
+                    koji_api.gssapi_login()
+                build = koji_api.getBuild(build_nvr, strict=True)
+
+            # Get the pullspec for the upstream equivalent
```

Comment:
```suggestion
            # Get the pullspec for the upstream equivalent
            # registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:v1.20.10-202310161945.el8.gdc4b478
```
Having an example value to look at would be helpful